packages/gatsby-plugin-cms/src/gatsby-node.ts

```diff
@@ -100,55 +105,74 @@ export const sourceNodes = async (
   gatsbyApi: SourceNodesArgs,
   options: Options
 ) => {
-  // Warning: Do not source remote and local nodes in a different order since this
-  // is important for the local nodes not to overrider remote ones
-  const nodes = await Promise.all([
-    fetchAllRemoteNodes(gatsbyApi, options),
-    fetchAllLocalNodes(gatsbyApi),
-  ]).then(([x, y]) => [...x, ...y].filter(Boolean))
+  const { webhookBody }: any = gatsbyApi
```

Comment:
Can this object be typed with a better type, can't?

Reply:
we can type it, but we can't guarantee the user passing this params, by default it's an open object

Reply:
Yeah, you're right. But at least could type something like `interface Interface { webhookBody?: any }`